### DISCUSSION TITLE
How set the Home Position

### DISCUSSION THREAD

**Discussion by Bersaker** [Jyers/Marlin#1018] [Q&A]
Hi.  Have a problem ... how or where can I change the home position?  I have to adjust the y axis + 2mm.  See image from 108.5 to 110.5 mm

Printer is a creality ender 3v2

![20210616_194323](https://user-images.githubusercontent.com/62776321/122268272-27c0b600-cedc-11eb-98af-f361a46d57d9.jpg)

**Comment by smooreace** [comment]
Thats because the "normal" dimensions arent correct! Measure your bed.

**Reply by Bersaker** [reply]
sorry.  I have made a mistake.  The bed isn't 220x220 ... it's 235x235 So 5 mm larger as value in template.

**Reply by Bersaker** [reply]
OK thanks.  I looked again and was able to correct the offset of the nozzle with m206.
